rsl-learning2022-Risheel/WIFIIndicator/app/src/main/res/drawable/round_button.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
```

Comment:
You should use oval share here to draw the circle.
Also i don't find any usage of this drawable anywhere.